lib/analytics.js

```diff
@@ -187,10 +187,13 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(
 
   // make ready callback
   var integrationCount = keys(integrations).length;
-  var ready = after(integrationCount, function() {
-    self._readied = true;
-    self.emit('ready');
-  });
+  var ready = function() {
+    self._callCount++;
```

Comment:
Can we make `callCount` a variable instead?

Reply:
done!